src/project/qml/MuseScore/Project/SaveToCloudDialog.qml

```diff
@@ -132,13 +151,32 @@ StyledDialogView {
             }
 
             RowLayout {
+                id: buttonsRow
                 Layout.alignment: Qt.AlignRight
                 spacing: 12
 
+                NavigationPanel {
+                    id: buttonsNavPanel
+                    name: "SaveToCloudOptions"
+                    enabled: buttonsRow.enabled && buttonsRow.visible
+                    direction: NavigationPanel.Horizontal
+                    section: root.navigationSection
+                    order: 2
+
+                    onNavigationEvent: function(event) {
+                        if (event.type === NavigationEvent.AboutActive) {
+                            event.setData("controlIndex", [saveButton.navigation.row, saveButton.navigation.column])
```

Comment:
We usually solve this problem (button activation priority when panel is activated) by setting correct buttons order
Save button must be first (order = 1)